ArjunaJTS/standalone/README.md

```diff
@@ -44,54 +44,40 @@ please override the setting in `start.sh` if it is incorrect.
 > If you are running on Windows you will need to port
   the script to Windows batch commands.
 
-Running with JacOrb:
+Running with openjdkorb:
 
 1. start a CORBA name server
 ```
-./start.sh jacorb NS
+./start.sh openjdkorb NS
```

Comment:
Do you think this argument should be required still? I can see it is read here: https://github.com/jbosstm/quickstart/blob/caef497c0b02ce985963de3a7ef64622edaf9416/ArjunaJTS/standalone/start.sh but I wonder if we should change that to not be needed?